### PR TITLE
Build WHL file

### PR DESCRIPTION
**Pull request by Rub21**
cc @rbavery

### CODE REVIEW

File: Dockerfile-cpu [hunk 1]
```diff
@@ -14,12 +14,14 @@ COPY ./deployment/config_decode.properties /home/model-server/config.properties
...
 
+WORKDIR src/sam_serve/
 COPY start ./
-
-WORKDIR /home/model-server
-COPY dist/sam_serve-1.0.0-py3-none-any.whl /home/model-server
-RUN pip install /home/model-server/sam_serve-1.0.0-py3-none-any.whl
+COPY pyproject.toml .
+COPY src/ src/
+COPY README.md .
```
Comment (rbavery) [inline Dockerfile-cpu:23]: do we need to copy this into the dockerfile?

File: Dockerfile-gpu [hunk 2]
```diff
@@ -14,12 +14,15 @@ COPY ./deployment/config_encode.properties /home/model-server/config.properties
 COPY ./deployment/append_memory_setting.sh /home/model-server/append_memory_setting.sh
 RUN chmod +x /home/model-server/append_memory_setting.sh \
     && /home/model-server/append_memory_setting.sh
+RUN pip install hatch
 
+WORKDIR /home/model-server
 COPY start ./
+COPY pyproject.toml .
+COPY src/ src/
+COPY README.md .
```
Comment (rbavery) [inline Dockerfile-gpu:23]: same question here

**Review by rbavery — COMMENTED** [review]
thanks Ruben! I like installing from the src with hatch like this. 


In the future maybe we can separate dev and prod images, where the Dockerfiles that run the services can instead install from pypi to simplify container setup.

https://github.com/developmentseed/segment-anything-services/pull/41

File: Dockerfile-cpu (hunk 1 above)
Comment (Rub21) [reply Dockerfile-cpu:23]: yes, it was necessary to run `hatch build`

**Comment by Rub21** [comment]
yes, i think that something that we can do in the GitHub actions the way that creates the images, dev and prod

**Review by rbavery — APPROVED** [review]

**Pull request closed without merging**